look-and-feel/react/src/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/no-array-index-key */
+import "@axa-fr/design-system-look-and-feel-css/dist/Skeleton/Skeleton.scss";
+
+export type SkeletonProps = {
+  className?: string;
+  grid: number[][];
+  ariaBusy?: boolean;
+  ariaLabel?: string;
+  maxCols?: number;
+  colGap?: number;
+  rowGap?: number;
+};
+
+export const Skeleton = ({
+  className = "af-skeleton",
+  grid = [],
+  ariaBusy = true,
+  ariaLabel = "Chargement",
+  maxCols = 12,
+  colGap = 1,
+  rowGap = 0.5,
+}: SkeletonProps) => (
+  <div
+    className={`${className}-container`}
+    aria-label={ariaLabel}
+    role="status"
+    aria-live="polite"
+    aria-busy={ariaBusy}
+    style={
+      {
+        "--max-cols": maxCols,
+        "--col-gap": `${colGap}rem`,
+        "--row-gap": `${rowGap}rem`,
+      } as React.CSSProperties
```

Comment:
je ne suis pas très fan car généralement quand on à des props lié à des tailles d'affichage elles sont en pixel.
alors mettre en rem ok mais il faut plutôt partir du principe que le colGap et en px et donc le diviser par 16 pour le mettre en rem.

exemple dans un img la props wigth est toujours exprimer en pixel.

Reply:
```suggestion
        "--col-gap": `calc(${colGap} / var(--font-size-base) * 1rem)`,
        "--row-gap": `calc(${rowGap} / var(--font-size-base) * 1rem)`,
      } as React.CSSProperties
```
je te propose cette version, de cette manière le dev fournit juste la valeur en pixel : 
```javascript
<Skeleton colGap={32} />
```

Reply:
A oui sa marche j'ai plutôt vue une truc du genre
```suggestion
        "--col-gap": `${colGap / 16}rem`,
        "--row-gap": `${rowGap / 16}rem`,
      } as React.CSSProperties
```
avec le 16 dans une const JS sinon sonar risque de pars être content ;) 
mais sa marche aussi

Reply:
non il vaut mieux une variable CSS car elle va surement être réutilisée dans d'autres variables CSS et puis c'est lié au style pas au JS

Reply:
c'est pas faux, ca me vas dans tous les cas